PortalFetch/crawler_download.py

```diff
@@ -204,6 +206,40 @@ def waitUtilPageLoaded(driver, count):
     raise ElementNotVisibleException("Could not load the full page!")
 
 
+def generateFilenameAndQuarter(quarterValue):
+    """Return quarter and fileName.
+
+    Args:
+        quarterValue:the quarter_value in crawler.config
+    Returns:
+        list of quarter str and fileName str
+
+    """
+    year = quarterValue[0:4]
+    resultList = []
+    quarterSwitcher = {
+        "1": "Summer",
+        "2": "Fall",
+        "3": "Winter",
+        "4": "Spring",
+    }
+    schoolSwitcher = {
+        "1": "Foothill",
+        "2": "De Anza",
+    }
+    school = schoolSwitcher.get(quarterValue[5], "")
+    quarter = quarterSwitcher.get(quarterValue[4], "")
+    if quarter == "Summer":
+        year = str(int(year)-1)
+    quarterOutput = year + " " + quarter + " " + school
+    resultList.append(quarterOutput)
+    if school == "De Anza":
+        school = "De_Anza"
+    fileNameOutput = year + "_" + quarter + "_" + school + "_courseData.json"
+    resultList.append(fileNameOutput)
+    return resultList
```

Comment:
instead of having 
```
resultList.append(quarterOutput)
resultList.append(fileNameOutput)
return resultList
```
You can just 
```
return quarterOutput, fileNameOutput
```